StudentDataAnalysator/ViewModels/MainViewModel.cs

```diff
@@ -160,10 +198,31 @@ private void ExecuteOpenFileDialog()
             SelectedPath = dialog.FileName;        
         }
 
-        public void ResetSelectedPath(string test)
+        public void SendList(string test)
         {
             if(SelectedPath != null)
-                SelectedPath = SelectedPath;
+            {
+                ExcelFileLoaderService _excelDataReader = new ExcelFileLoaderService(SelectedPath);
+
+                if (_excelDataReader.GetTableType() == (int)TableTypeEnum.StudentsResultTable)
+                    StudentsList = StudentsList;
+                else
+                    LogsList = LogsList;
+            }
+        }
+
+        private void GetExcelData(string path)
+        {
+            ExcelFileLoaderService _excelDataReader = new ExcelFileLoaderService(path);
+
+            if (_excelDataReader.GetTableType() == (int)TableTypeEnum.StudentsResultTable)
```

Comment:
Try to simplify this if statement by extracting the condition in another method.